ha/delete-cluster.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+virsh shutdown node01
+virsh shutdown node02
+virsh shutdown node03
```

Comment:
Let's go with `virsh destroy` here, as we don't care about cleanly shutting down the nodes. Also if the shutdown fails then the `virsh undefine` will also fail, leaving the nodes around.